apps/preview/nuxt/pages/examples/SfTextarea.vue

```diff
@@ -11,7 +11,16 @@
       >
         {{ label }}
       </span>
-      <SfTextarea v-bind="state" v-model="value" class="w-full" />
+      <SfTextarea
+        v-bind="state"
+        v-model="value"
+        :class="[
+          'w-full',
+          {
+            '!bg-disabled-100 !ring-disabled-300 !ring-1 !text-disabled-500': disabled || readonly,
```

Comment:
Shouldn't read-only state be styled with neutral classes instead of disabled?

Reply:
Ok, right, text should be neutral. Thx.